src/applications/vaos/new-appointment/components/ReviewPage/SelectedProviderSection.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import newAppointmentFlow from '../../newAppointmentFlow';
+
+import { LANGUAGES } from '../../../utils/constants';
+
+export default function SelectedProviderSection({ data, vaCityState }) {
+  const provider = data.communityCareProvider;
+  const hasProvider =
+    !!provider && !!Object.keys(data.communityCareProvider).length;
+
+  return (
+    <div className="vads-l-grid-container vads-u-padding--0">
+      <div className="vads-l-row vads-u-justify-content--space-between">
+        <div className="vads-u-flex--1 vads-u-padding-right--1">
+          <h3 className="vaos-appts__block-label">Preferred provider</h3>
+          {!hasProvider && <>No provider specified</>}
```

Comment:
Observation: I haven't seen the angle brackets used like this before, what is the reasoning? `<>No provider specified</>`

Reply:
@LanceGundersen Those brackets are a shorthand for `<React.Fragment>` (see https://reactjs.org/docs/fragments.html). I think @jbalboni is using it here to avoid having to wrap the "No provider specified" in an element such as a `<span>`.  Alternatively, we could also do `{!hasProvider && 'No provider specified'}` here.

Reply:
Yeah, I think I had a `<br/>` tag in here at one point, and that's why it uses `<></>` instead of a string.